mod/reportedcontent/views/default/reportedcontent/css.php

```diff
@@ -4,3 +4,6 @@
 .elgg-icon-report-this:before {
 	content: "\f071";
 }
+.elgg-menu-extras .elgg-menu-item-report-this .elgg-anchor-label {
+	display: none;
```

Comment:
By inaccessible do you mean ARIA? We could set the width to 0 instead of changing display.